### ISSUE TITLE
Unable to reproduce the reported EvalGEN results with the official checkpoint

### ISSUE THREAD

**Issue by wj-sim** [roy-ch/Dual-Data-Alignment#15]
Hello, thank you for releasing the code, checkpoint, and EvalGEN dataset.

I am trying to reproduce the EvalGEN results for DDA reported in Table 10 of the paper. Using the newly released official checkpoint and the first generated image (index 0) for each EvalGEN prompt, I obtain an average accuracy of 94.4%, while the paper reports 97.2%.

## Versions

- Repository commit: `8b9c06e75e63f4688bc25ac43a7e3412878cf67f`
- Python: 3.10
- PyTorch: 2.4.1+cu121
- torchvision: 0.19.1+cu121
- Pillow: 11.1.0
- libjpeg: 9.0
- xFormers: not installed; DINOv2 uses its fallback attention implementation

Official checkpoint:

```text
DDA_ckpt.pth
SHA256: b27a31d39374803ddeff02bfabb2be76e190b04300490cddfafb24f683f37e3e
```

This hash matches the current Hugging Face checkpoint.

## Evaluation subset

Following Appendix E, I selected the first generated image (index 0) for every prompt:

```python
paths = [
    path for path in all_generator_images
    if path.name.lower().endswith("_0.jpg")
]
```

This gives exactly:

```text
Flux:      553 images
GoT:       553 images
Infinity:  553 images
NOVA:      553 images
OmniGen:   553 images
Total:   2,765 images
```

## Model loading

```python
model = DINOv2ModelWithLoRA(
    name="dinov2_vitl14",
    lora_rank=8,
    lora_alpha=1,
    lora_targets=None,
)

checkpoint = torch.load("DDA_ckpt.pth", map_location="cpu")
model.load_state_dict(checkpoint["model"], strict=True)
model.to(device)
model.eval()
```

The checkpoint loads with `strict=True` without missing keys, unexpected keys, or shape mismatches.

## Input preprocessing

I followed `Inference/inference.py` exactly:

```python
test_transform = transforms.Compose([
    transforms.CenterCrop(336),
    transforms.ToTensor(),
    transforms.Normalize(
        mean=[0.48145466, 0.4578275, 0.40821073],
        std=[0.26862954, 0.26130258, 0.27577711],
    ),
])

image = Image.open(image_path).convert("RGB")
input_tensor = test_transform(image)
```

I did not apply additional JPEG encoding, resizing, blur, or other corruption because the released EvalGEN images are already stored as JPEG QF96.

The DataLoader uses:

```python
DataLoader(
    dataset,
    batch_size=64,
    shuffle=False,
    num_workers=8,
    pin_memory=True,
)
```

## Prediction and metric

All EvalGEN images are fake, with label 1:

```python
with torch.no_grad():
    scores = model(images).sigmoid().flatten()
    predictions = (scores >= 0.5).long()
```

For each generator:

```python
accuracy = (predictions == 1).float().mean()
```

The final result is the equally weighted arithmetic mean of the five generator accuracies.

## Results

| Generator | My result | Table 10 |
|---|---:|---:|
| Flux | 81.37 | 89.9 |
| GoT | 99.28 | 99.5 |
| Infinity | 95.12 | 97.8 |
| NOVA | 98.55 | 99.5 |
| OmniGen | 97.83 | 99.5 |
| Mean | 94.43 | 97.2 |

The discrepancy is particularly large for Flux. Since I am using the newly released official checkpoint, the Appendix E evaluation subset, and the preprocessing implemented in the released inference code, I would appreciate any clarification on whether there are additional evaluation details required to reproduce the Table 10 results.

Thank you for your help.
